raiden_contracts/tests/fixtures/base/web3_fixtures.py

```diff
@@ -68,3 +75,19 @@ def auto_revert_chain(web3: Web3):
     snapshot_id = web3.testing.snapshot()
     yield
     web3.testing.revert(snapshot_id)
+
+
+def _call_and_transact(
+        contract_function: ContractFunction,
+        transaction_params: Optional[Dict] = None,
+) -> str:
+    """ Executes contract_function.{call, transaction}(transaction_params) and returns txhash """
+    # First 'call' might raise an exception
+    contract_function.call(transaction_params)
+    return contract_function.transact(transaction_params)
+
+
+@pytest.fixture(scope='session')
+def call_and_transact():
```

Comment:
Since this will be used extensively in tests, it would be acceptable to use `autouse=True` for it and not explicitly add it to the list of fixtures for each test.

Reply:
I'll do this in the PR.

Reply:
Done.